### PR TITLE
Update lod_tensor.rst

### PR DESCRIPTION
**Pull request by tink2123**
- Modified some inaccurate descriptions
- Added the complete code example

### CODE REVIEW

File: doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst [hunk 1]
```diff
@@ -153,14 +153,25 @@ LoD-Tensor
 
 recursive_seq_lens 是一个双层嵌套列表，也就是列表的列表，最外层列表的size表示嵌套的层数，也就是lod-level的大小；内部的每个列表，对应表示每个lod-level下，每个元素的大小。
 
-.. code-block :: text
+.. code-block:: python
 
+  #创建lod-tensor
+  import paddle.fluid as fluid
```
Comment (wzzju) [inline doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst:159]: `import numpy as np`

File: doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst [hunk 2]
```diff
@@ -153,14 +153,25 @@ LoD-Tensor
...
+                                    [1],[1],[1],[1],
+                                    [1],
+                                    [1],[1],
+                                    [1],[1],[1]]).astype('int64') ,
+                            [[3,1,2] , [3,2,4,1,2,3]],
+                            fluid.CPUPlace())
+  
   #查看lod-tensor嵌套层数
-  print len(recursive_seq_lengths)
+  print len(a.recursive_seq_lengths())
```
Comment (wzzju) [inline doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst]: `print len(a.recursive_sequence_lengths())` is right.

File: doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst [hunk 3]
```diff
@@ -153,14 +153,25 @@ LoD-Tensor
...
   # output：2
 
   #查看最基础元素个数
-  print sum(recursive_seq_lengths[-1])
+  print sum(a.recursive_seq_lengths()[-1])
```
Comment (wzzju) [inline doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst]: `print sum(a.recursive_sequence_lengths()[-1])` is right.

File: doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst [hunk 4]
```diff
@@ -262,17 +235,63 @@ y_d 也为一个LoDTensor：
...
 输出结果为：
 
-.. code-block :: python
+.. code-block:: text
+
+  array([[1.1],[2.2],[3.3],[4.4],[2.2],[3.3],[4.4],[2.2],[3.3],[4.4]])
+
+**查看序列长度**
+
+可以通过查看序列长度得到 LoDTensor 的层级信息：
```
Comment (wzzju) [inline doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst]: 可以通过查看序列长度得到 LoDTensor 的递归序列长度。层级信息应该是LoD吧？请参考[paddle.fluid.create_lod_tensor](http://paddlepaddle.org/documentation/docs/zh/1.2/api_cn/fluid_cn.html#paddle.fluid.create_lod_tensor)的介绍。

Comment (tink2123) [reply doc/fluid/user_guides/howto/basic_concept/lod_tensor.rst]: All of them have been  fixed. Many thanks!

**Review by wzzju — APPROVED** [review]
LGTM! # @shanyi15 Please have a look.

**Review by shanyi15 — APPROVED** [review]
LGTM

**Pull request merged**